benchmarks/pom.xml

```diff
@@ -36,6 +36,11 @@
             <artifactId>ballerina</artifactId>
             <type>zip</type>
         </dependency>
+        <dependency>
+            <groupId>com.github.chewiebug</groupId>
+            <artifactId>gcviewer</artifactId>
```

Comment:
Do not use this unless you have an approval from project leads.

Reply:
Will finalize this after discussing.